src/components/Chat/Chat.jsx

```diff
@@ -1,23 +1,32 @@
-import { useSelector } from 'react-redux';
-import { getCountryName } from '@/redux-store/selectors.js';
-import { ChatStyled, Header, HeaderContent, MessageBlock } from './ChatStyled';
-
+import logo from '@/images/logo.svg';
+import ChatHeader from '../ChatHeader/ChatHeader';
+import MessageList from '../MessageList/MessageList';
 import MessageBar from '../MessageBar/MessageBar';
+import {
+  ChatStyled,
+  MessageBlock,
+  NoMassegesNotification,
+  Logo,
+} from './ChatStyled';
 
-const Chat = () => {
-  const countryName = useSelector(getCountryName);
-
-  return (
-    <ChatStyled>
-      <Header>
-        <HeaderContent>
-          <h5>{countryName || 'Country Name'}</h5>
-        </HeaderContent>
-      </Header>
-      <MessageBlock />
-      <MessageBar />
-    </ChatStyled>
-  );
-};
-
+const Chat = ({ countryData, setCountryData, setSubscriptionCountryRooms }) => (
+  <ChatStyled>
+    <ChatHeader countryData={countryData} />
+    <MessageBlock>
+      {countryData?.country?.groupMessages.length ? (
+        <MessageList countryData={countryData} />
+      ) : (
+        <NoMassegesNotification>
+          <Logo src={logo} alt="logo" />
```

Comment:
https://www.w3schools.com/tags/att_img_width.asp#:~:text=Tip%3A%20Always%20specify%20both%20the,the%20appropriate%20space%20to%20it.

Reply:
Done